src/libtools/calculator.cc

```diff
@@ -20,57 +20,51 @@
 namespace votca {
 namespace tools {
 
-void Calculator::UpdateWithDefaults(Property &options, std::string package) {
-
-  // copy options from the object supplied by the Application
-  std::string id = Identify();
-  Property options_id = options.get("options." + id);
-
-  // add default values if specified in VOTCASHARE
+std::string Calculator::GetVotcaShare() {
```

Comment:
can you make the function static, because it does not need members.

Reply:
sure, I missed that ;)